mk/dev.mk

```diff
@@ -182,6 +186,21 @@ dev/install/goimports: ## Bootstrap: Install goimports
 dev/install/helm3: ## Bootstrap: Install Helm 3
 	curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
 
+.PHONY: dev/install/helm-docs
+dev/install/helm-docs: ## Bootstrap: Install helm-docs
+	@if [ -e $(HELM_DOCS_PATH) ]; then echo "Helm Docs $$( $(HELM_DOCS_PATH) version ) is already installed at $(HELM_DOCS_PATH)" ; fi
```

Comment:
@nickolaev, `helm-docs` default command is to regenerate readmes with the default option.So, doing this step again will change the docs. Is this ok for now? or I can fix this and then we can merge